<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmMunicipio</class>
 <widget class="QDialog" name="frmMunicipio">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>338</width>
    <height>346</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TopoDelProp-Municipio</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>361</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Seleccione el municipio:</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>301</width>
     <height>131</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtMunicipio">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>301</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="bttAceptar">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>310</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Aceptar</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidgetSrc">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>301</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>361</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Seleccione el SRC del municipio:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
